### PR TITLE
making MainWin easier to read by extracting methods

### PR DESCRIPTION
**Pull request by mbed67**
@SunyataZero and @yigitsever this PR is an experiment aimed at making the code more readable by extracting parts of it to methods with a descriptive name. I think it will still allow for your idea Yigit, to have more generic functions such as `create_gui_element`. I'm open to suggestions for improvement or change.

### CODE REVIEW

File: mc/gui/main_win.py [hunk 1]
```diff
@@ -27,39 +27,58 @@ class MainWin(QtWidgets.QMainWindow):
...
+        self.setStyleSheet("selection-background-color:#bfef7f; selection-color:#000000;")
+
+    def set_up_main_container(self):
         central_w2 = QtWidgets.QWidget()
         self.setCentralWidget(central_w2)
         hbox_l3 = QtWidgets.QHBoxLayout()
         central_w2.setLayout(hbox_l3)
+        return hbox_l3
 
+    def add_first_panel_to_main_container(self, hbox_l3):
```
Comment (yigitsever) [inline mc/gui/main_win.py]: Asking due to inexperience, do we need to specify hbox_13 & vbox_14 or can we use more general names?

File: mc/gui/main_win.py [hunk 2]
```diff
@@ -72,43 +91,49 @@ def __init__(self):
...
-        self.on_breathing_settings_changed()
-        self.update_rest_timer()
+    def configure_breathing_settings(self, vbox_l4):
+        vbox_l4.addWidget(self.br_settings_wt)
+        self.br_settings_wt.updated_signal.connect(self.on_breathing_settings_changed)
+        self.br_settings_wt.breathe_now_button_clicked_signal.connect(self.open_breathing_dialog)
 
-        # Setup of Systray
-        self.setup_systray()
+    def set_window_title_with_version_and_storage(self):
```
Comment (yigitsever) [inline mc/gui/main_win.py]: maybe just set_window_title, title may not use version and storage information in the future

File: mc/gui/main_win.py [hunk 3]
```diff
@@ -27,39 +27,58 @@ class MainWin(QtWidgets.QMainWindow):
...
 
+    def add_first_panel_to_main_container(self, hbox_l3):
+        vbox_l4 = self.new_panel_in_main_window(hbox_l3)
+        self.add_and_configure_active_breathing_phrase_to_panel(vbox_l4)
+        vbox_l4.addWidget(self.breathing_history_wt)
 
-        vbox_l4 = QtWidgets.QVBoxLayout()
-        hbox_l3.addLayout(vbox_l4)
-        self.active_breathing_phrase_qgb = QtWidgets.QGroupBox(self.tr("Active Breathing Phrase"))
+    def add_and_configure_active_breathing_phrase_to_panel(self, vbox_l4):
```
Comment (yigitsever) [inline mc/gui/main_win.py]: maybe add_active_breathing phrase and configure_active breathing phrase

Or add_active_breathing_phrase with optional arguments?

File: mc/gui/main_win.py (hunk 1 above)
Comment (SunyataZero) [reply mc/gui/main_win.py]: Thank you for the question, `hbox_lx` stands for:
* h: horizontal
* box
* l: layout (can also be w if it's a widget)
* x: level in the layout

We could optionally add more descriptive information in addition to this though, maybe at the start, something like:

`descriptivename_hboxlx`

What do you think? @yigitsever  @mbed67

Comment (yigitsever) [reply mc/gui/main_win.py]: I'm for it, hbox_l3 (not 13 :) is too specific in my opinion

**Comment by SunyataZero** [comment]
@mbed67 @yigitsever Maybe we can prefix all the setup/init functions with a desciptive word? This would help readers separate these functions out from the others. (In this case i am assuming that they are setup functions that we only want to be run from the __init__ function

File: mc/gui/main_win.py (hunk 1 above)
Comment (mbed67) [reply mc/gui/main_win.py]: I'm also in favor of adding a descriptive name while also keeping the specification. I will change it accordingly.

**Comment by mbed67** [comment]
@SunyataZero @yigitsever 

> Maybe we can prefix all the setup/init functions with a desciptive word? 

I agree that this would make it more clear for readers. What do you think of '_setup_' as a prefix?

**Comment by SunyataZero** [comment]
> What do you think of '_setup_' as a prefix?

I like it! :+1:

**Comment by yigitsever** [comment]
> I agree that this would make it more clear for readers. What do you think of '_setup_' as a prefix?

yup, even complies with #107!

**Pull request merged**